### PR TITLE
Suggested fixes by ShellCheck

### PR DESCRIPTION
**Pull request by enoch85**
Not tested cc @ezraholm50

### CODE REVIEW

**Comment by enoch85** [comment]
@ezraholm50 Can you please merge all your changes into this PR and I'll check it later today. Thanks!

File: static/change_mysql_pass.sh [hunk 1]
```diff
@@ -3,13 +3,13 @@
 # Tech and Me, ©2016 - www.techandme.se
 
 SHUF=$(shuf -i 17-20 -n 1)
-NEWMYSQLPASS=$(cat /dev/urandom | tr -dc "a-zA-Z0-9@#*=" | fold -w $SHUF | head -n 1)
+NEWMYSQLPASS=$(cat < /dev/urandom | tr -dc "a-zA-Z0-9@#*=" | fold -w "$SHUF" | head -n 1)
```
Comment (enoch85) [inline static/change_mysql_pass.sh]: Shoulnd't this be a '>' instead?

Comment (ezraholm50) [inline static/change_mysql_pass.sh]: Nope said it should be this way, thought it was strange to, but after changing the error went away so, we'll see.

**Comment by enoch85** [comment]
This PR failed. ShellCheck are wrong, we are right. WIll close.

**Pull request closed without merging**